CMakeLists.txt

```diff
@@ -1,34 +1,63 @@
 cmake_minimum_required(VERSION 3.17)
 cmake_policy(SET CMP0100 NEW)  # handle .hh files
-project(CLAP_HELPERS C CXX)
+project(clap-helpers C CXX)
 
-add_library(clap-helpers INTERFACE)
-target_include_directories(clap-helpers SYSTEM INTERFACE include)
-target_link_libraries(clap-helpers INTERFACE clap)
+option(CLAP_HELPERS_BUILD_TESTS "Build tests for the CLAP Helper" FALSE)
 
-# CLAP_BUILD_TESTS is inherited from clap
-if (${CLAP_BUILD_TESTS})
-    enable_testing()
-    find_package(Catch2 3 QUIET)
+add_library(${PROJECT_NAME} INTERFACE)
+target_include_directories(${PROJECT_NAME} INTERFACE include)
 
-    if (Catch2_FOUND)
-        add_executable(clap-helpers-tests EXCLUDE_FROM_ALL
+if (NOT TARGET clap)
+    message(STATUS "${PROJECT_NAME}: Downloading CLAP using CPM")
+
+    message(STATUS "${PROJECT_NAME}: including CPM")
+    include(cmake/external/CPM.cmake)
+    CPMAddPackage(
+            NAME "clap"
+            GITHUB_REPOSITORY "free-audio/clap"
+            GIT_TAG "next"
```

Comment:
this will be set to `"1.2.0"` once that one is ready, right?

Reply:
yup